umbraco-cloud/upgrades/version-specific-upgrades/migrate-from-umbraco-7-to-8.md

```diff
@@ -0,0 +1,161 @@
+---
+
+# Migrate from Umbraco 7 to Umbraco 8 on Umbraco Cloud
+
+Taking your Umbraco CMS project from Umbraco 7 to 8 is called a migration as it requires that the data is migrated in the process. This article covers each step involved in this process.
+
+Read the [general article about Content migration](https://docs.umbraco.com/umbraco-cms/fundamentals/setup/upgrading/version-specific/migrate-content-to-umbraco-8.md#what-are-the-limitations) to learn more about limitations and other things related to migrating your Umbraco site from 7 to 8.
+
+## Video tutorial
+
+<iframe width="800" height="450" title="Migrate an Umbraco Cloud project from 7 to 8: Introduction" src="https://www.youtube.com/embed/wNIDdgdAt8s?rel=0" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+
+You can find the full playlist here: [Migrate an Umbraco Cloud project from 7 to 8](https://www.youtube.com/playlist?list=PLgX62vUaGZsGceCXveMkprlDV5a3K94db)
+
+## Prerequisites
+
+* A Umbraco 7 Cloud project running **the latest version of Umbraco 7**.
+* Make sure Umbraco Forms data is not handled as content.
+  * See [Umbraco Forms on Cloud](../../deployment/umbraco-forms-on-cloud.md#how-forms-are-handled-on-umbraco-cloud) for more details on how to check the setting.
+* A clean Cloud project running the latest version of Umbraco 8 with **at least 2 environments**.
+
+{% hint style="info" %}
+We strongly recommend having at least 2 environments on the Umbraco 8 project.
+
+Should something fail during the migration, the Development environment can always be removed and re-added in order to start over on a clean slate.
+{% endhint %}
+
+## Step 1: Content migration
+
+1. Clone down the Umbraco 7 Cloud project.
+2. Run the project locally and **restore** Content and Media.
+3. Clone down the Development environment from the Umbraco 8 Cloud project.
+4. Install the [ProWorks Umbraco 8 Migrations](https://www.nuget.org/packages/ProWorks.Umbraco8.Migrations) community package on the cloned Umbraco 8 site.
+5. Copy `~/App_Data/Umbraco.sdf` or `~/App_Data/Umbraco.mdf` from the cloned Umbraco 7 project.
+6. Paste the file into `~/App_Data` on the clone of the Umbraco 8 project.
+7. Open `web.config` from the Umbraco 8 project.
+8. Locate the `Umbraco.Core.ConfigurationStatus` key.
+9. Replace the value with the version your Umbraco 7 project is running.
+10. Run the Umbraco 8 project locally
+11. Authorize the migration - Cloud credentials are used for this.
+
+![Authorize upgrade](../images/upgrade-to-8_1.png)
+
+12. Click **Continue** to start the migration.
+13. Log in to the backoffice and verify that everything is there once the migration is complete.
+
+{% hint style="info" %}
+If your login does not work, try the following approach:
+
+1, Copy the `UsersMembershipProvider` attributes from your Umbraco 7 `web.config`, to the Umbraco 8 `web.config`.
+2. Try to login again.
```

Comment:
The list items are displayed as a sentence on GitBook instead of list. Not sure if it's because it's in a Note.

Reply:
Hm. Yeah! It might be. I don't think this needs a list anyways, so I'll rewrite it a bit.